js/private/js_info.bzl

```diff
@@ -4,8 +4,8 @@ JsInfo = provider(
     doc = "Encapsulates information provided by rules in rules_js and derivative rule sets",
     fields = {
         "declarations": "A depset of declaration files produced by the target",
-        "npm_linked_packages": "A depset of files in npm linked package dependencies of the target and the target's transitive deps",
-        "npm_package_store_deps": "A depset of NpmPackageStoreInfo providers from non-dev npm dependencies of the target and the target's transitive dependencies to use as direct dependencies when linking downstream npm_package targets with npm_link_package",
+        "npm_deps": "A depset of files in npm dependencies of the target and the target's transitive deps",
```

Comment:
Why add the term `deps` here while removing it elsewhere? None of the other fields here use the term after changing `npm_package_store_*` from "deps" to "infos"...

Reply:
It felt out of place on `npm_package_store_deps`. More reasoning on that here: https://github.com/aspect-build/rules_js/pull/1620

`npm_linked_packages` doesn't seem like the right name either since the depset includes both linked package and transitive deps in the packages store.

Might be able to find something better than `npm_deps`. I'll open up another PR and we can bikeshed there.